snuba/datasets/transactions_processor.py

```diff
@@ -77,6 +77,15 @@ def process_message(self, message, metadata=None) -> Optional[ProcessedMessage]:
             processed["start_ts"], processed["start_ms"] = self.__extract_timestamp(
                 data["start_timestamp"],
             )
+            status = transaction_ctx.get("status", None)
```

Comment:
I believe it's worth writing a `2` explicitly and not relying on a ClickHouse default.